containers/codespaces-linux/.devcontainer/base.Dockerfile

```diff
@@ -70,13 +70,14 @@ RUN apt-get update && export DEBIAN_FRONTEND=noninteractive \
     && mkdir -p /usr/local/etc/vscode-dev-containers/ \
     && mv -f /tmp/scripts/first-run-notice.txt /usr/local/etc/vscode-dev-containers/
 
-# Install Python, PHP, Ruby utilities
+# Install Python, PHP, Ruby utilities, and JupyterLab
 RUN bash /tmp/scripts/python-debian.sh "none" "/opt/python/latest" "${PIPX_HOME}" "${USERNAME}" "true" \
     # Install rvm, rbenv, any missing base gems
     && chown -R ${USERNAME} /opt/ruby/* \
     && bash /tmp/scripts/ruby-debian.sh "none" "${USERNAME}" "true" "true" \
     # Link composer
     && ln -s $(which composer.phar) /usr/local/bin/composer \
+    && bash /tmp/scripts/jupyterlab-debian.sh \
```

Comment:
Would be great to add a line the test script under `/containers/codespaces-linux` to validate that we installed jupityer and it's available.  

That test script is run as the "smoke test" CI on the is PR. The PR is expected to fail right now due to some unresolved oryx issues, but you'll at least be able to see the result of any lines you add

Reply:
@joshspicer Thanks! I removed the jupyter-debian.sh and just do the `pip install ...` from the base.Dockerfile now.